Drift/Kernel.php

```diff
@@ -79,6 +86,9 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
     protected function configureRoutes(RouteCollectionBuilder $routes): void
     {
         $confDir = $this->getApplicationLayerDir().'/config';
-        $routes->import($confDir.'/routes.yml');
```

Comment:
Another discussion here.

Multiple routes depending on the environment can be achieved in a less magical way by using different bundles (contexts). From an architectural perspective, including testing and extension, this makes more sense.